datadog/dogshell/monitor.py

```diff
@@ -148,14 +163,41 @@ def _file_post(cls, args):
     def _update(cls, args):
         api._timeout = args.timeout
         format = args.format
-        options = None
+
+        to_update = {}
+        if args.type:
+            if args.type_opt:
+                msg = '[WARNING] Duplicate arguments for `type`. Using optional value --type'
+                print(msg)
```

Comment:
For printing, let's use the `print_err` utility function that you can import from `datadog.dogshell.common` https://github.com/DataDog/datadogpy/blob/9a463de4769dd3661007d0dae9bd37918d182d28/datadog/dogshell/common.py#L11-L16, and use the format `"WARNING: "` for consistency with how we print other warnings, as in https://github.com/DataDog/datadogpy/blob/9a463de4769dd3661007d0dae9bd37918d182d28/datadog/dogshell/common.py#L35

Reply:
This is all done and tests adjusted accordingly